definitions/ext-nas/summary_metrics.yml

```diff
@@ -0,0 +1,33 @@
+# Tested on Netgear ReadyNAS
+ipAddress:
+  title: IP Address
+  unit: STRING
+  tag:
+    key: device_ip
+
+model:
+  title: Model
+  unit: STRING
+  query:
+    select: latest(SysDescr)
```

Comment:
Wondering if this should also be a tag on the entity. Since it seems to make sense as one.

Reply:
There are times when the SysDesc of a device can be several hundred characters in length. Without being able to predict this field, I moved away from that idea.

Reply:
wow. reasonable enough 😅